types/time.go

```diff
@@ -2577,3 +2578,40 @@ func DateFSP(date string) (fsp int) {
 	}
 	return
 }
+
+var (
+	dateRange = []gotime.Time{
+		gotime.Date(1000, 1, 1, 0, 0, 0, 0, gotime.Local),
```

Comment:
Are they the same with https://github.com/pingcap/tidb/blob/master/types/time.go#L112?

Reply:
Fixed.